packages/npm-resolver/src/index.ts

```diff
@@ -265,7 +265,10 @@ function pickMatchingLocalVersionOrNull (
     case 'version':
       return versions[spec.fetchSpec] ? spec.fetchSpec : null
     case 'range':
-      return semver.maxSatisfying(localVersions, spec.fetchSpec, true)
+      return semver.maxSatisfying(localVersions, spec.fetchSpec, {
+        includePrerelease: true,
```

Comment:
I will probably have to revert this option. Also, it appears that the same options should be suplied to 

https://github.com/pnpm/pnpm/blob/01c1a55aeccf0f106b98a6122af5f256a68419f3/packages/supi/src/install/index.ts#L528